ros/openapi/openapi.json

```
@@ -624,6 +627,36 @@
                         ]
                     }
                 }
+            },
+            "filterOSParam": {
+                "name": "os",
+                "in": "query",
+                "required": false,
+                "description": "Filter system with the given os version",
```

```suggestion
                "description": "Filter system with the given RHEL version",
```
As you are listing RHEL versions` values and not full version names, will it be good to indicate it in description?